src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+import React from 'react';
+import _ from 'underscore';
+//import MicroscopyMetadataTool from "4dn-microscopy-metadata-tool";
+import { console, object, layout } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { ItemFileAttachment } from './components/ItemFileAttachment';
+import DefaultItemView from './DefaultItemView';
+
+
+export default class MicroscopeConfigurationView extends DefaultItemView {
+
+    getTabViewContents(){
+        const initTabs = [];
+        initTabs.push(MicroMetaTabView.getTabObject(this.props));
+        return initTabs.concat(this.getCommonTabs()); // Add remainder of common tabs (Details, Attribution)
+    }
+
+}
+
+
+
+
+function onLoadSchema(complete) {
```

Comment:
This and the 2 similar functions will be removed and replaced with class methods that should POST/PATCH/GET from portal endpoint(s).

Requirements TBD.